install.php

```diff
@@ -1653,6 +1654,7 @@ function process_form(the_form)
 		<div id="brdtitle" class="inbox">
 			<h1><span><?php echo $lang_install['FluxBB Installation'] ?></span></h1>
 			<div id="brddesc"><p><?php echo $lang_install['FluxBB has been installed'] ?></p></div>
+			<?php header("Location: index.php"); ?>
```

Comment:
Attempts to send HTTP headers mid-way through the HTML body will fail miserably in multiple cases, for example, if output buffering is disabled.

Reply:
Seemed to be fine under testing. Perhaps the body can not be outputted since nobody would see it anyway?

Reply:
Try disabling the `output_buffering` in PHP. [^1]

[^1]: https://www.php.net/manual/en/outcontrol.configuration.php

Reply:
Ah, in that case would it simply be better to simply disable the output of the installed page entirely?

Reply:
> Ah, in that case would it simply be better to simply disable the output of the installed page entirely?

No, the only correct way is to send headers BEFORE any output, i.e. within the top `<?php ... ?>` code block.

Reply:
That's sort of what I meant. If there is no other output than the header, then it is before all other output.